connect/client.go

```diff
@@ -106,6 +109,53 @@ type restClient struct {
 	client    httpClient
 }
 
+func (rs *restClient) GetFiles(itemUUID string, vaultUUID string) ([]onepassword.File, error) {
+	span := rs.tracer.StartSpan("GetFiles")
+	defer span.Finish()
+
+	jsonURL := fmt.Sprintf("/v1/vaults/%s/items/%s/files", vaultUUID, itemUUID)
+	request, err := rs.buildRequest(http.MethodGet, jsonURL, http.NoBody, span)
+	if err != nil {
+		return nil, err
+	}
+	response, err := rs.client.Do(request)
+	if err != nil {
+		return nil, err
+	}
+	if err := expectMinimumConnectVersion(response, version{1, 3, 0}); err != nil {
+		return nil, err
+	}
+	var files []onepassword.File
+	if err := parseResponse(response, http.StatusOK, &files); err != nil {
+		return nil, err
+	}
+
+	return files, nil
+}
+
+func (rs *restClient) DownloadFile(fileUUID, itemUUID, vaultUUID, targetDirectory string) error {
+	file, err := rs.GetFile(fileUUID, itemUUID, vaultUUID)
+	if err != nil {
+		return err
+	}
+	content, err := rs.GetFileContent(file)
+	if err != nil {
+		return err
+	}
+
+	dst, err := os.Create(filepath.Join(targetDirectory, filepath.Base(file.Name)))
```

Comment:
This creates a file with `0666` (rw for all) permissions [by default](https://pkg.go.dev/os#Create). I am not sure if that is a sane default in this case. `0600` (rw for the current user only) sounds safer to me. Can we make that the default and/or make the permissions configurable?

Reply:
Thanks for the suggestion! Just added this too.